### DISCUSSION TITLE
How to install lts version of NPM

### DISCUSSION THREAD

**Discussion by sandorduba** [coreybutler/nvm-windows#658] [Support] [answered]
Hi,
Using the nvm install command I can install a specific version of node, which comes with a specific version of npm.

I would like to use the LTS version of both, but using the LTS version of the node the NPM version is not the latest long-term supported. So I want to install that separately.

I was not able to install the LTS version of NPM using `npm i -g npm@lts`:
```
PS C:\Users\Sandor_Duba> npm i -g npm@lts
npm ERR! code EEXIST
npm ERR! path C:\Program Files\nodejs\npm.cmd
npm ERR! Refusing to delete C:\Program Files\nodejs\npm.cmd: is outside C:\Program Files\nodejs\node_modules\npm and not a link
npm ERR! File exists: C:\Program Files\nodejs\npm.cmd
npm ERR! Remove the existing file and try again, or run npm
npm ERR! with --force to overwrite files recklessly.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Sandor_Duba\AppData\Roaming\npm-cache\_logs\2021-09-07T11_35_46_563Z-debug.log
```

Any suggestion on this?

**Comment by coreybutler** [comment, marked as answer]
It looks like you may be unintentionally using an older version of Node. Remember to run `nvm use x.x.x`  where `x.x.x` is the version you want to use. See https://github.com/coreybutler/nvm-windows/issues/300#issuecomment-332523342, which covers the exact error message you're seeing.

**Reply by coreybutler** [reply]
@sandorduba - in re-reviewing what your wrote, I noticed your username has a space in it. This was the most likely cause of the issue. This has been a long-standing issue that was just resolved. Upgrading to v1.1.8 will resolve the problem.
